vmod/vmod_std.vcc

```diff
@@ -576,12 +576,16 @@ Example::
 		std.set_ip_tos(0);
 	}
 
-$Restrict client
+$Restrict vcl_deliver vcl_hit vcl_miss vcl_pass vcl_pass vcl_purge vcl_recv vcl_synth vcl_backend_error vcl_backend_response
```

Comment:
Should we support a syntax similar to bits parameters?

```
$Restrict client backend -vcl_recv -vcl_backend_fetch
```

Reply:
I think I like the idea of not having to write all the vcl subroutines but one, but it can also be confusing, what if we have: `$Restrict -vcl_recv`, does it mean we allow everything but `vcl_recv` ? Or would it be necessary to use one of `client/backend/housekeeping` with the `-` syntax ?

Reply:
No, I think we should just treat a minus prefix as clearing the relevant bit(s) and adding a "negative" note to the documentation. I don't think vmodtool cares about what we tells `$Restrict` today and will happily compile a VMOD restricted to `vcl_non_existent`.